### PR TITLE
Windows application driver fix

### PR DESCRIPTION
**Pull request by nicerloop**
Two fixes to streamline the use of Windows Application Driver aka WinAppDriver aka WAD:
- move supporting dlls from System folder next to WinAppDriver.exe: otherwise the supporting dlls are not in path and WAD fails to run
- set APPIUM_WAD_PATH environment variable with custom WAD location: appium-windows-driver can use WAD as installed by the installer, but needs to be informed of the location of the scoop installed version

These two fixes allow to transparently replace WAD installed by the installer with WAD installed by scoop.

- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/windows-application-driver.json [hunk 1]
```diff
@@ -6,6 +6,10 @@
     "url": "https://github.com/Microsoft/WinAppDriver/releases/download/v1.2.1/WindowsApplicationDriver_1.2.1.msi",
     "hash": "a76a8f4e44b29bad331acf6b6c248fcc65324f502f28826ad2acd5f3c80857fe",
     "bin": "Windows Application Driver\\WinAppDriver.exe",
+    "post_install": "mv $dir\\System\\* $dir\\Windows*",
```
Comment (HUMORCE) [inline bucket/windows-application-driver.json:9]:
```suggestion
    "post_install": [
        "Move-Item \"$dir\\System\\*\", \"$dir\\Windows Application Driver\\*\" -Destination \"$dir\"",
        "Remove-Item \"$dir\\System\", \"$dir\\Windows Application Driver\""
    ],
```

Move all files to the app root directory.

File: bucket/windows-application-driver.json [hunk 2]
```diff
@@ -6,6 +6,10 @@
...
+    "env_set": {
+        "APPIUM_WAD_PATH": "$dir\\Windows Application Driver\\WinAppDriver.exe"
+    },
```
Comment (HUMORCE) [inline bucket/windows-application-driver.json:12]:
```suggestion
```
I think users should set it up manually when using it with appium. At least, the envvar`APPIUM_WAD_PATH` have not been created when install via its MSI installer.

Comment (nicerloop) [reply bucket/windows-application-driver.json:12]: I will replace with a note.

File: bucket/windows-application-driver.json (hunk 1 above)
Comment (nicerloop) [reply bucket/windows-application-driver.json:9]: This will conflict with the creation of a bin shim. I will remove the bin shim.

**Review by HUMORCE — APPROVED** [review]

**Comment by nicerloop** [comment]
I cannot install windows-application-driver with both the moved files and the bin shim:

Creating shim for 'WinAppDriver'.
Can't shim 'WinAppDriver.exe': File doesn't exist.

That is why I proposed to remove the bin shim, which is not necessary as the actual location is to be expose as an environment variable to Appium.

**Comment by HUMORCE** [comment]
> Creating shim for 'WinAppDriver'.
Can't shim 'WinAppDriver.exe': File doesn't exist.

Fixed: https://github.com/ScoopInstaller/Main/commit/04607fabbecad5664e78e65bef455cb238a7b275

**Pull request merged**